src/widgets/current-refinements/__tests__/current-refinements-test.ts

```diff
@@ -158,31 +132,32 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/current-ref
 
       helper.addFacetRefinement('facet', 'facet-val1');
 
-      widget.init({
-        helper,
-        createURL: () => '#cleared',
-        instantSearchInstance: {},
-      });
+      widget.init!(
+        createInitOptions({
+          helper,
+        })
+      );
 
       const renderParameters = {
-        results: {
-          facets: [
-            {
-              name: 'facet',
-              data: {
-                'facet-val1': 1,
+        results: new SearchResults(helper.state, [
```

Comment:
I am missing something with the `SearchResults` constructor. How do we pass the counts for the facets? (you can see that they're missing in the snapshots, but not in the stories)

Reply:
the counts are taken from the second response, bu there you are only having one, and seems to be a wrong facets interface

`[ { facets: { xxx: 5 }}, { facets: { xxxTaken: 5000 }}]`